Libs/MRML/Core/vtkMRMLViewNode.cxx

```diff
@@ -242,6 +249,13 @@ double* vtkMRMLViewNode::defaultBackgroundColor2()
   return backgroundColor2;
 }
 
+//------------------------------------------------------------------------------
+double* vtkMRMLViewNode::defaultBoxColor()
+{
+  static double boxColor[3] = {1.0, 0.0, 1.0};
```

Comment:
Having a static member variable in a method is very obscure and could have unintended side effects. Also, naming of the method does not follow VTK conventions. Please change it to:

```
void vtkMRMLViewNode::GetDefaultBoxColor(double color[3])
{
  color[0] = 1.0;
  color[1] = 0.0;
  color[2] = 1.0;
}
```

I know that `defaultBackgroundColor` and `defaultBackgroundColor2` does not follow the naming conventions either but we'll fix that later.

Reply:
Done